systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -1069,6 +1075,499 @@ void testRestoreOffsetsInConsumerGroup(ExtensionContext extensionContext) {
+                .withUsername(kafkaUserSourceName)
+                .withPasswordSecret(passwordSecretSource)
+            .endKafkaClientAuthenticationScramSha512()
+            .withNewTls()
+                .withTrustedCertificates(certSecretSource)
+            .endTls()
+            .addToConfig("config.storage.replication.factor", -1)
+            .addToConfig("offset.storage.replication.factor", -1)
+            .addToConfig("status.storage.replication.factor", -1)
+            .addToConfig("checkpoints.topic.replication.factor", "1")
+            .addToConfig("config.storage.replication.factor", 1)
+            .addToConfig("emit.checkpoints.enabled", "true")
+            .addToConfig("emit.checkpoints.interval.seconds", "1")
+            .addToConfig("heartbeats.topic.replication.factor", "1")
+            .addToConfig("offset.storage.replication.factor", 1)
+            .addToConfig("offset-syncs.topic.replication.factor", "1")
+            .addToConfig("refresh.groups.interval.seconds", "1")
+            .addToConfig("replication.factor", "1")
+            .build();
+
+        KafkaMirrorMaker2ClusterSpec targetClusterWithScramSha512Auth = new KafkaMirrorMaker2ClusterSpecBuilder()
+            .withAlias(kafkaClusterTargetName)
+            .withBootstrapServers(KafkaResources.tlsBootstrapAddress(kafkaClusterTargetName))
+            .withNewKafkaClientAuthenticationScramSha512()
+                .withUsername(kafkaUserTargetName)
+                .withPasswordSecret(passwordSecretTarget)
+            .endKafkaClientAuthenticationScramSha512()
+            .withNewTls()
+                .withTrustedCertificates(certSecretTarget)
+            .endTls()
+            .addToConfig("config.storage.replication.factor", -1)
+            .addToConfig("offset.storage.replication.factor", -1)
+            .addToConfig("status.storage.replication.factor", -1)
+            .addToConfig("checkpoints.topic.replication.factor", "1")
+            .addToConfig("config.storage.replication.factor", 1)
+            .addToConfig("emit.checkpoints.enabled", "true")
+            .addToConfig("emit.checkpoints.interval.seconds", "1")
+            .addToConfig("heartbeats.topic.replication.factor", "1")
+            .addToConfig("offset.storage.replication.factor", 1)
+            .addToConfig("offset-syncs.topic.replication.factor", "1")
+            .addToConfig("refresh.groups.interval.seconds", "1")
+            .addToConfig("replication.factor", "1")
+            .build();
+
+        resourceManager.createResource(extensionContext, KafkaMirrorMaker2Templates.kafkaMirrorMaker2(testStorage.getClusterName(), kafkaClusterTargetName, kafkaClusterSourceName, 1, true)
+            .editSpec()
+                .withClusters(targetClusterWithScramSha512Auth, sourceClusterWithScramSha512Auth)
+            .endSpec()
+            .build());
+
+        LOGGER.info("[DEFAULT] Setting topic to {}, cluster to {} and changing user to {} for next internalKafkaClient.",
```

Comment:
`DEFAULT` is little bit confusing - it's default like default settings, default logging or what?

Reply:
Original idea was to show which type of authentication method is currently used. Tls/Scram.. But after separation of tests it makes almost no sense now.
Removed these prefixes.